setup.py

```diff
@@ -25,7 +25,6 @@
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
-        'License :: OSI Approved :: License (ISCL)',
```

Comment:
I have very little knowledge about licences. Care to explain the difference between GPLv3 vs AGPLv3? I usually go for GPLv3 mostly because I cant be bothered reading everything